test/utils/image/manifest.go

```diff
@@ -212,6 +212,8 @@ const (
 	VolumeRBDServer
 	// WindowsServer image
 	WindowsServer
+	// SnapshotController image
+	SnapshotController
```

Comment:
Is this used anywhere?

Reply:
Removed this. It was originally for testing with an unreleased version of the snapshot-controller.